packages/cli/src/wrapper.ts

```diff
@@ -11,7 +11,7 @@ import { lintConfigCallback } from './commands/lint';
 import type { CommandOptions } from './types';
 
 export function commandWrapper<T extends CommandOptions>(
-  commandHandler: (argv: T, config: Config, version: string) => Promise<void>
+  commandHandler?: (argv: T, config: Config, version: string) => Promise<void>
```

Comment:
I'd better go with a dummy command handler (e.g. `() => {}`) instead of making it optional.

Reply:
Works both ways, changed to suggested.